### PR TITLE
Add logarithmic scale option to Statistics and History graph cards

### PR DESCRIPTION
**Pull request by wittypluck**
## Proposed change

This PR adds documentation for the new "logarithmic scale" option in the Statistics and History graph cards.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
https://github.com/home-assistant/frontend/pull/18637

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by wittypluck** [comment]
Hi,
I'm not sure why lint gives me the following error. 
Could somebody help on this ?
Thanks,

`error  Incorrect usage of the term: “axis”, use “Axis” instead  terminology`

File: source/_dashboards/history-graph.markdown [hunk 1]
```diff
@@ -48,6 +48,11 @@ show_names:
   description: If false, no entity names are shown in the card.
   type: boolean
   default: true
+logarithmic_scale:
+  required: false
+  description: If true, numerical values on the vertical axis will be displayed with a logarithmic scale.
```
Comment (FletcherAU) [inline source/_dashboards/history-graph.markdown]:
```suggestion
  description: If true, numerical values on the Y-axis will be displayed with a logarithmic scale.
```

File: source/_dashboards/statistics-graph.markdown [hunk 2]
```diff
@@ -64,6 +64,11 @@ hide_legend:
   description:  If true, the legend will be hidden.
   type: boolean  
   default: false
+logarithmic_scale:
+  required: false
+  description: If true, numerical values on the vertical axis will be displayed with a logarithmic scale.
```
Comment (FletcherAU) [inline source/_dashboards/statistics-graph.markdown]:
```suggestion
  description: If true, numerical values on the Y-axis will be displayed with a logarithmic scale.
```

**Review by FletcherAU — CHANGES_REQUESTED** [review]
Convention appears to be `Y-axis` over `vertical axis` ie, https://github.com/home-assistant/home-assistant.io/blob/86cef97aeedd5b67242d0d1f929a20a465c36a3b/source/_dashboards/sensor.markdown?plain=1#L60

(This may also help with the linter problem though that's only a guess)

**Comment by FletcherAU** [comment]
Looks like that's solved it :)

**Comment by wittypluck** [comment]
Thank you for the suggestion!

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @wittypluck and @FletcherAU :+1:

**Pull request merged**